### ISSUE TITLE
Legal documents before going live

### ISSUE THREAD

**Issue by brainless** [brainless/dwata#167]
As a user I would like to have access to these documents to be able to use the product:

## Needed by Google OAuth2 app and Razor Pay:
- Privacy policy

## Needed by Google OAuth2 app:
- Terms of service

## Needed by Razor Pay
- Adhaar / Passport / Voter ID
- Company PAN
- Certificate of Incorporation

**Comment by brainless** [comment]
Here is a Privacy Policy generated by https://mydukaan.io/tools/privacy-policy-generator:
- [Privacy_Policy.md](https://github.com/user-attachments/files/17148375/Privacy_Policy.md)

**Comment by brainless** [comment]
I found another generator: https://termly.io/products/privacy-policy-generator/

This one is very elaborate and asks detailed questions about what information we collect from users but I get the feeling that none of it applies to us since we, the company, do not collect anything. dwata runs on user's device and does not send anything to our company. Does GDPR apply in such cases?

**Comment by HilaryTorn** [comment]
No, it doesn't.  We are using Plausible for website analytics and it is the same case.  It doesn't collect that data, doesn't store cookies, so no consent is needed for the website. 

So I want to take some time to figure out what we actually do need, or figure out how to say "we don't collect anything" and what actually is needed there if we aren't collecting data.

**Comment by HilaryTorn** [comment]
@brainless I checked and the privacy policy for the web vs app is the same. Plus, we aren't collecting or selling personal data on either atm. So I would use this: https://pixlie.com/privacy/

I just added: We do not collect any data for our Desktop app Dwata.

When we give the option to collect data, we can update it here.
